src/app/assinante/service/assinante.service.ts

```diff
@@ -0,0 +1,35 @@
+import { HttpClient, HttpHeaders } from '@angular/common/http';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/PadawansTeam/artmais-frontend/pullRequest?prid=8399665)
```suggestion
import { HttpClient, HttpHeaders } from "@angular/common/http";
```